src/default-xpring-client.ts

```diff
@@ -177,11 +178,9 @@ class DefaultXpringClient implements XpringClientDecorator {
     return new GetTxResponseWrapper(getTxResponse)
   }
 
-  // TODO(keefertaylor): Add tests for this method once send is hooked up. 
-  private async getFee(): Promise<XRPDropsAmount> {
-    const getFeeRequest = new GetFeeRequest()
+  private async getMinimumFee(): Promise<XRPDropsAmount> {
```

Comment:
is `getMinimumFee` used right now? Maybe `git` is just making the diff look weird.

Reply:
Not yet. We're building toward #136 (implement send) which needs to get a fee and generate a last ledger sequence number (which requires the current ledger sequence number) in order to generate a transaction. 

In general, these helper methods are just mirroring the existing helper methods in `default-xpring-client`: https://github.com/xpring-eng/Xpring-JS/blob/master/src/legacy/legacy-default-xpring-client.ts#L200. There are differences in implementation in both that we're using a different set of protocol buffers and we implemented a slightly different subset of RPCs in rippled (they functionally serve the same set of data though).